website-frontend/src/lib/models/student_council.ts

```diff
@@ -0,0 +1,7 @@
+import { object, string, type InferOutput } from 'valibot';
+
+export const StudentCouncil = object({
+	flexible_content: string()
```

Comment:
This flexible content from the WYSIWIG editor will be Raw HTML that needs to be sanitized. Use the `cleanHtml` validator from `../models-helper.ts`